ts_src/index.ts

```diff
@@ -93,24 +97,42 @@ export function mnemonicToSeed(
 }
 
 export function mnemonicToEntropy(
-  mnemonic: string,
+  mnemonic: string | Buffer,
   wordlist?: string[],
 ): string {
   wordlist = wordlist || DEFAULT_WORDLIST;
   if (!wordlist) {
     throw new Error(WORDLIST_REQUIRED);
   }
 
-  const words = normalize(mnemonic).split(' ');
+  const mnemonicAsBuffer = Buffer.from(
+    normalize(typeof mnemonic === 'string' ? mnemonic : mnemonic.toString()),
```

Comment:
@Gudahtt Turns out we need to make sure that the mnemonic string is properly normalized before we (re)cast it to buffer.

Reply:
This might defeat the purpose of the patch. The purpose was to prevent the mnemonic from being represented in memory as a single string at any point.

Perhaps we can make this a documented requirement of passing in a `Buffer`? Or maybe there's a way to perform this normalization directly on the `Buffer`.

Reply:
I don't believe we can normalize directly on the Buffer. I think we'll have to make it a documented requirement.